### PR TITLE
Elvish: new package at 0.14.1

### PR DESCRIPTION
**Pull request by monarrk**
Elvish combines usable features of regular shells and syntax and features which make programming in it feel like using any other scripting language. It also has other useful programming features, such as a module system, command line syntax highlighting, and a powerful editing library.

## Installed manifest of package

```
/var/db/kiss/installed/elvish/version
/var/db/kiss/installed/elvish/sources
/var/db/kiss/installed/elvish/manifest
/var/db/kiss/installed/elvish/depends
/var/db/kiss/installed/elvish/checksums
/var/db/kiss/installed/elvish/build
/var/db/kiss/installed/elvish/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
```

## New package

- [ * ] Latest upstream version.
- [ * ] I agree to maintain this package.
- [ * ] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

**Comment by dylanaraps** [comment]
The manifest is incorrect. No files are installed at all?

File: community/elvish/build [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -e
+
+:> nostrip
+
+make
```
Comment (dylanaraps) [inline community/elvish/build:5]: You must also run `make install` with `DESTDIR` set to `$1`.

**Comment by monarrk** [comment]
because it's written in go, it installs to `$HOME/go/bin` or `$GOPATH` which I don't believe is picked up by the manifest. I was willing to leave it that way but if there's a way to make it install to /usr/bin I'd of course be willing to

**Comment by dylanaraps** [comment]
Yes. 

```sh
install -Dm755 elvish/elvish "$1/usr/bin/elvish"
```

Something like this is required if the Makefile does not support it.

**Comment by dylanaraps** [comment]
If the Makefile puts it in `$HOME`, don't run the Makefile.

**Comment by dylanaraps** [comment]
Note: The package manager is useless if packages go installing things in `$HOME` during builds (it is bypassed entirely).

**Comment by monarrk** [comment]
Ok, I just updated the build to use `go build` directly rather than make. here's the new manifest:
```/var/db/kiss/installed/elvish/version
/var/db/kiss/installed/elvish/sources
/var/db/kiss/installed/elvish/manifest
/var/db/kiss/installed/elvish/depends
/var/db/kiss/installed/elvish/checksums
/var/db/kiss/installed/elvish/build
/var/db/kiss/installed/elvish/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/bin/elvish
/usr/bin/
/usr/```

**Comment by dylanaraps** [comment]
Nice :+1:

**Pull request merged**